internal/report/table/table.go

```diff
@@ -148,3 +148,21 @@ func statusSymbol(passed bool) string {
 
 	return color.GreenString("✔")
 }
+
+func calDuration(results []report.TestResult) time.Duration {
+	if len(results) == 0 {
+		return 0
+	}
+	start := results[0].StartTime
+	end := results[0].EndTime
+	for _, r := range results {
+		if start.After(r.StartTime) {
+			start = r.StartTime
+		}
+		if end.Before(r.EndTime) {
+			end = r.EndTime
+		}
+	}
```

Comment:
I find this a tiny, tiny bit more legible and concise:

```suggestion
	start := time.Now()
	end := start
	for _, r := range results {
		if r.StartTime.Before(start) {
			start = r.StartTime
		}
		if r.EndTime.After(end) {
			end = r.EndTime
		}
	}
```

Reply:
genius!